packages/jaeger-ui/src/components/TracePage/CriticalPath/index.test.js

```diff
@@ -22,22 +22,14 @@ import sanitizeOverFlowingChildren from './utils/sanitizeOverFlowingChildren';
 import test6 from './testCases/test6';
```

Comment:
these imports are not needed?

Reply:
But we want to test them right.  Without importing how can we do that?

Reply:
not the tests, but the two implementation functions that you're no longer calling

Reply:
I couldn't comment on the previous line

Reply:
Okk I got it!